code/web/src/modules/user/Profile.js

```diff
@@ -25,11 +26,11 @@ import user from '../../setup/routes/user';
 // Component
 const Profile = (props) => {
   let button;
-  if (props.user.isEditMode) {
-    button = <Button onClick={() => props.changeEditMode(props.user)} theme="secondary">Save Profile</Button>
-  } else {
-    button = <Button onClick={() => props.changeEditMode(props.user)} theme="secondary">Edit Profile</Button>  
-  }
+  // if (props.user.isEditMode) {
+  //   button = <Button onClick={() => props.changeEditMode(props.user)} theme="secondary">Save Profile</Button>
+  // } else {
+    // button = <Button onClick={() => props.changeEditMode(props.user)} theme="secondary">Edit Profile</Button>  
+  // }
```

Comment:
I think we decided to get rid of these as well, so maybe pull these comments out too?